Sources/Login/LoginCoordinator.swift

```diff
@@ -92,7 +95,16 @@ final class LoginCoordinator: NSObject,
                     root.pushViewController(networkErrorScreen, animated: true)
                 }
             }
+
         root.setViewControllers([rootViewController], animated: true)
+        if let startupError {
```

Comment:
Wondering if we could give this a more descriptive name just so it's clear they user has been unable to either get the id token or verify it?